composer.json

```diff
@@ -71,7 +71,7 @@
     "codeception/module-laravel5": "^1.1",
     "codeception/module-rest": "^1.2",
     "codeception/module-webdriver": "^1.0",
-    "fzaninotto/faker": "^1.9",
+    "fzaninotto/faker": "master@dev",
```

Comment:
This is just a workaround because it's abandoned and doesn't formally support PHPv8

Reply:
Faker is only really used by our Seeder system, so I guess we just need to know that it works there?